articles/ros2_dds_security.md

```diff
@@ -125,72 +125,73 @@ Let's delve further into these.
 
-##### Support security files lookup methods
+``` shell
+export ROS_SECURITY_ROOT_DIRECTORY="/home/bob/.ros/sros2_keystore"
+ros2 run <package> <executable> --ros-args --enclave="/front/camera"
+```
 
-If using the directory tree approach to organize security files, RCL supports two different methods for looking up a given node instance's security files in the tree:
-
-- **Exact**: Only load security files from a directory exactly matching the fully-qualified name of the node instance.
-For example, given a node named "baz_123" within the "/foo/bar/" namespace, only load security files from `<root>/foo/bar/baz_123/`.
-This is the default behavior.
-- **Prefix**: Attempt to load the most specific set of security files, but if they can't be found, check for security files under a less-specific node name.
-For example, given a node named "baz_123" within the "/foo/bar/" namespace, load security files from `<root>/foo/bar/baz_123/`.
-However, if that directory doesn't exist, find the most specific (i.e. longest) node name that _does_ have security files within that namespace (e.g. `<root>/foo/bar/baz_12/`, or `<root>/foo/bar/baz/`, etc.).
-Note that it will not search higher in the namespace hierarchy.
-
-The desired lookup method can be specified by setting the `$ROS_SECURITY_LOOKUP_TYPE` environment variable to "MATCH_EXACT" (case-sensitive) for the **Exact** method, or "MATCH_PREFIX" (case-sensitive) for the **Prefix** method.
+#### Manual specification
 
+RCL supports specifying the path to a directory containing the set of security files for the process that needs to be launched.
+The set of files expected within that directory are the same as outlined in the "Directory tree of all security files" section above for individual enclave directories.
 
-#### Manual specification
+This can be specified by setting the `ROS_SECURITY_DIRECTORY_OVERRIDE` environment variable to point to the directory containing the security files.
+Note that this setting takes precedence over `ROS_SECURITY_ROOT_DIRECTORY` with `--enclave`.
 
-RCL supports specifying the path to a directory containing the set of security files for the exact node instance that needs to be launched.
-The set of files expected within that directory are the same as outlined in the "Directory tree of all security files" section above for individual node instance directories.
+Note the following two examples load from the same enclave path as demonstrated prior:
 
-This can be specified by setting the `$ROS_SECURITY_NODE_DIRECTORY` environment variable to point to the directory containing the security files.
-Note that this setting takes precedence over `$ROS_SECURITY_ROOT_DIRECTORY`.
+``` shell
+export ROS_SECURITY_DIRECTORY_OVERRIDE="/home/bob/.ros/sros2_keystore/enclaves/front/camera"
+ros2 run <package> <executable>
+```
 
+``` shell
+export ROS_SECURITY_ROOT_DIRECTORY="/dev/null"
```

Comment:
moving tihs line in the example above would make 1 example showing overriding the root directory and one showing the overriding of the CLA

Reply:
Done: ea400f6